src/game/requester.h

```diff
@@ -4,11 +4,12 @@
 
 #include <stdint.h>
 
-void Requester_Init(REQUEST_INFO *req);
-void Requester_Remove(REQUEST_INFO *req);
+void Requester_Init(REQUEST_INFO *req, const uint16_t num_items);
```

Comment:
In this context, `const` is unnecessary – it indicates that the value of `num_items` will not be modified within the function body, however, since this is only a function declaration, there is no function body to consider. Because of this, it's a common practice to avoid declaring primitive types as `const` in header files. (Please note that pointers are a different story – declaring them as `const` in declarations promises that the function body won't alter the content they point to, and that is a useful promise.)